CHANGELOG.md

```diff
@@ -19,6 +19,10 @@
 - More debug logs for UIViewController tracing (#4942)
 - Avoid creating unnecessary User Interaction transactions (#4957)
 
+### Internal
+
+- Change parameter of `SDKInfo.initWithOptions` to be nullable (#4968)
```

Comment:
`m`: I would maybe describe this as a fix. This could lead to a rare crash if the SDK wasn't initialized properly or something.

Reply:
Sounds good to me.